.buildkite/pipeline.yml

```diff
@@ -396,9 +396,9 @@
     - echo "--- Building py37-cpu Ray image for the test."
     - LINUX_WHEELS=1 ./ci/ci.sh build
     - pip install -q docker
-    - python ci/build/build-docker-images.py --py-versions py37 --device-types cpu --build-type LOCAL --build-base
+    - python ci/build/build-docker-images.py --py-versions py36 --device-types cpu --build-type LOCAL --build-base
```

Comment:
Was this on purpose? The comment in the echo statement above still says py37 (which probably makes more sense given that py36 is already EOL)

Reply:
> which probably makes more sense given that py36 is already EOL

Ah, thanks for the context.

The CI environment currently uses py36, so I changed the the image to py36 for Ray client compatibility.

A better move would probably be to upgrade the CI environment for this job to py37 instead. I will look into doing that.

Reply:
Nice catch on the echo line -- we should keep that consistent.

Reply:
cc @simon-mo for thoughts/context on our use of Python 3.6 in the Ray CI.

Reply:
Fixed to use Py37 in the test image and in the build step's host CI environment.